### PR TITLE
Bugfix/add link to big number

### PR DESCRIPTION
**Pull request by shekinah-smith**
Added a self-invoking function to attach `ALAMODE_CHARTS` to the window when Alamode is called. The other way, was creating a new object every time which meant if there were multiple big number values that needed to have a link added, they weren't getting attached properly. 

Corresponds with: https://github.com/mode/webapp/pull/9885

### CODE REVIEW

File: alamode.js [hunk 1]
```diff
@@ -4,6 +4,9 @@
 var version = "0.23";
 
 var alamode = {
+  createAlamodeChartsWindowObj: (function() {
```
Comment (shekinah-smith) [inline alamode.js]: I'm a little iffy on this but seems like the best to way to attache the object to the window without it creating a new object every time.

Comment (shekinah-smith) [reply alamode.js]: It should only call this once, no matter how many time `addLinkToBigNumber` is called.

File: alamode.js [hunk 2]
```diff
@@ -2257,7 +2260,6 @@ var alamode = {
   },
 
   addLinkToBigNumber: function(o) {
```
Comment (benmyerson) [inline alamode.js]: You could do 
`window["ALAMODE_CHARTS"] = window["ALAMODE_CHARTS"] || {};`
here if you weren't thrilled about the self-invoking function.

Comment (shekinah-smith) [reply alamode.js]: Oh true! Thank you!

**Pull request merged**